ca_municipalities/people.py

```diff
@@ -0,0 +1,162 @@
+                            row[key] = corrections(row[key])
+                        elif row[key] in corrections:
+                            row[key] = corrections[row[key]]
+
+                    organizationClassification = row.get('classification')
+                    organizationClassification = self.validateOrganizationClassification(organizationClassification)
+
+                    organization = None
+                    organizationName = row['organization']
+                    if organizationName:
+                        if self.organizations.get(organizationName.lower()):
+                            organization = self.organizations.get(organizationName.lower())
+                        else:
+                            organization = Organization(organizationName, classification=organizationClassification)
+                            organization.add_source(self.csv_url)
+                            yield organization
+                            self.organizations[organizationName] = organization
+                    else:
+                        continue
+
+                    # ca_qc_laval: "maire et president du comite executif", "conseiller et membre du comite executif"
+                    # ca_qc_montreal: "Conseiller de la ville; Membre…", "Maire d'arrondissement\nMembre…"
+                    if row.get('primary role'):
+                        row['primary role'] = re.split(r'(?: (?:et)\b|[;\n])', row['primary role'], 1)[0].strip()
+                    else:
+                        row['primary role'] = 'Councillor'
+
+                    roleName = row.get('primary role')
+
+                    post = Post(role=roleName, label=organizationName, organization_id=organization._id)
+                    yield post
+
+                    if row.get('full name') and not re.search(r'[a-z]', row['full name']):
+                        row['full name'] = re.sub(r'(?<=\b[A-Z])[A-ZÀÈÉ]+\b', lambda x: x.group(0).lower(), row['full name'])
+
+                    name = '{}'.format(row['full name']).strip(' .,')
+
+                    role = row['primary role']
+
+                    # ca_qc_laval: "maire …", "conseiller …"
+                    if role not in ('candidate', 'member') and not re.search(r'[A-Z]', role):
+                        role = role.capitalize()
+
+                    if self.district_name_format_string:
+                        if row['district id']:
+                            district = self.district_name_format_string.format(**row)
+                        else:
+                            district = self.jurisdiction.division_name
+                    elif row.get('district name'):
+                        district = row['district name']
+                    elif self.fallbacks.get('district name'):
+                        district = row[self.fallbacks['district name']] or self.jurisdiction.division_name
+                    else:
+                        district = self.jurisdiction.division_name
```

Comment:
Assuming the spreadsheet contains a 'district name' column, this can be simplified to `district = row['district name']`

Reply:
Done. Thanks